cognite/extractorutils/configtools/elements.py

```diff
@@ -545,6 +562,12 @@ def stop_pushers(self) -> None:
 
 
 class ConfigType(Enum):
+    """
+    Enum to define if the configuration file should be fetched remotely.
+
+    Valid values: local|remote
```

Comment:
I prefer for these to not be in docs, they are always visible from the code, and we run the risk of them falling out of sync.
``` suggestion
```